src/index/IndexImpl.h

```diff
@@ -343,6 +343,13 @@ class IndexImpl {
         return std::nullopt;
       case Datatype::TextRecordIndex:
         return getTextExcerpt(id.getTextRecordIndex());
+      case Datatype::WordVocabIndex:
+        auto result = _textVocab.indexToOptionalString(id.getVocabIndex());
+        if (result.has_value() && result.value().starts_with(VALUE_PREFIX)) {
+          result.value() =
+              ad_utility::convertIndexWordToValueLiteral(result.value());
+        }
```

Comment:
This code is not needed for the `WordVocabIndex`, and also this function doesn't exist anymore.
In the one place where you use it you have to probably access the `TextVocab` for now.